tests/Logic/TestUsersHelper.php

```diff
@@ -157,4 +157,61 @@ public function test_too_long_unused_nicename() {
 		$should_also_be_shorter_and_different = UsersHelper::get_unused_nicename( $long_nicename );
 		$this->assertTrue( strlen( $should_also_be_shorter_and_different ) <= $max_length );
 	}
+
+	/**
+	 * Test that create_or_get_user no longer falls back to looking for users by other fields
```

Comment:
Does it make sense to have "no longer" in a test? After some time has passed any reference to when "no longer" was is gone. What do you think? Could we have the comment just explain what we are testing in the test?

Reply:
Makes sense. I pushed a fix. What do you think?